packages/gatsby-plugin-google-gtag/src/gatsby-browser.js

```diff
@@ -18,13 +18,15 @@ exports.onRouteUpdate = ({ location }) => {
     window.gtag(`event`, `page_view`, { page_path: pagePath })
   }
 
+  const { delayOnRouteUpdate = 0 } = pluginOptions
```

Comment:
This is incorrect as `delayOnRouteUpdate` is set on `pluginOptions.pluginConfig`, see https://github.com/Simon-Tang/gatsby/blob/master/packages/gatsby-plugin-google-gtag/src/gatsby-ssr.js#L12

Reply:
Thanks @LekoArts, good catch! Amended to fix it.